api/src/main/resources/Reference_Application_Concepts-23.xml

```diff
@@ -115,7 +115,15 @@
   <concept concept_id="162821" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-06 22:32:39.0" version="" changed_by="1" date_changed="2015-01-06 22:32:47.0" retired_by="1" uuid="162821AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="162822" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-06 22:33:22.0" version="" changed_by="1" date_changed="2015-01-06 22:33:29.0" retired_by="1" uuid="162822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="163139" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-12-13 15:00:18.0" version="" changed_by="1" date_changed="2015-12-13 15:02:03.0" retired_by="1" uuid="163139AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="165095" retired="false" datatype_id="3" class_id="7" is_set="false" creator="1" date_created="2018-04-18 14:34:20.0" version="" changed_by="1" date_changed="2018-04-18 14:34:30.0" retired_by="1" uuid="165095AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept_numeric concept_id="1343" units="cm" precise="true"/>
+  <concept_numeric concept_id="5085" hi_absolute="250.0" low_absolute="0.0" units="mmHg" precise="false"/>
+  <concept_numeric concept_id="5086" hi_absolute="150.0" low_absolute="0.0" units="mmHg" precise="false"/>
+  <concept_numeric concept_id="5087" hi_absolute="230.0" low_absolute="0.0" units="beats/min" precise="false"/>
+  <concept_numeric concept_id="5088" hi_absolute="43.0" low_absolute="25.0" units="DEG C" precise="true"/>
```

Comment:
According to this commit https://github.com/openmrs/openmrs-module-referencemetadata/commit/9f0a937e2d7975b5edd0a842b4f085db0d20f6d1 the concept numeric rows should be entered in different file names.

Reply:
From the output that I had got from the data exchange module. Everything in the 3 XML files have been generated by the module ( including the dates and anything else)